tests/integration_tests/data_adapter/test_data_adapter_on_sg.py

```diff
@@ -0,0 +1,169 @@
+import os
+import numpy as np
+
+from data_gradients.managers.detection_manager import DetectionAnalysisManager
+from data_gradients.managers.segmentation_manager import SegmentationAnalysisManager
+from data_gradients.managers.classification_manager import ClassificationAnalysisManager
+
+from super_gradients import setup_device
+from super_gradients.training.dataloaders.dataloaders import coco2017_val, cityscapes_stdc_seg50_val, cifar10_val
+from super_gradients.training.utils.collate_fn import (
+    DetectionDataloaderAdapter,
+    SegmentationDataloaderAdapter,
+    ClassificationDataloaderAdapter,
+)
+
+
+def test_python_detection():
```

Comment:
Tests are not membest or TestCase class. Was it intentional?

Reply:
Not sure what to do with these tests.
I am testing Roboflow datasets, for the sake of using using adatper on non-sg datasets, but this requires external datasets.
Downloading the datasets would slow down the tests by a lot, I really don't think we want that.

So until now I only used it manually, but it's also very sad to not add these tests to TestCase.
What do you think should be the right way ? Just replacing these datasets by dummy datasets ?

Reply:
I think using tinycoco dataset that we already have and implementing a few custom classes that return samples in a variety of formats would be great way to test this.